cppcache/integration-test/ThinClientFailoverInterest2.hpp

```diff
@@ -289,7 +289,7 @@ const char* vals[] = {"Value-1", "Value-2", "Value-3", "Value-4"};
 const char* nvals[] = {"New Value-1", "New Value-2", "New Value-3",
                        "New Value-4"};
 
-const char* regionNames[] = {"DistRegionAck", "DistRegionNoAck"};
+const char* regionNames[] = {"DistRegionAck", "not-used"};
```

Comment:
Same

Reply:
Good catch. I put in the "not-used" as a baby step, but forget to convert to scalar in a few files.